heron/api/src/java/BUILD

```diff
@@ -6,6 +6,7 @@ load("/tools/rules/heron_deps", "heron_java_proto_files")
 
 api_deps_files =  \
     heron_java_proto_files() + [
+        ":classification",
```

Comment:
@billonahill - this might not work since there is no target defined in this file?